packages/devextreme/testing/tests/DevExpress.ui.widgets/progressBar.tests.js

```diff
@@ -201,6 +201,43 @@ QUnit.module('states', {
 });
 
 QUnit.module('aria accessibility', () => {
+    QUnit.test('aria-readonly is removed in initialization', function(assert) {
+        const $element = $('#progressbar').dxProgressBar({
+            readOnly: true,
+        });
+
+        assert.equal($element.attr('aria-readonly'), undefined, 'aria-readonly is removed');
```

Comment:
fix everywhere please
```suggestion
        assert.strictEqual($element.attr('aria-readonly'), undefined, 'aria-readonly is removed');
```

Reply:
[ab1c9f1](https://github.com/DevExpress/DevExtreme/pull/27984/commits/ab1c9f1c45526c7aeaee0d7864c1d5a772f50a40)